samples/contrib/aws-samples/README.md

```diff
@@ -6,132 +6,22 @@ This folder contains many example pipelines which use [AWS SageMaker Components
-          app.kubernetes.io/name: pipelines-runner
-          app.kubernetes.io/part-of: kubeflow
-          app.kubernetes.io/version: 0.2.0
-        name: pipeline-runner
-        namespace: kubeflow
-        resourceVersion: "11787"
-        selfLink: /api/v1/namespaces/kubeflow/serviceaccounts/pipeline-runner
-        uid: d86234bd-7fa5-11ea-a8f2-02934be6dc88
-      secrets:
-      - name: pipeline-runner-token-dkjrk
-      ``` 
-**Option 2** Store the IAM credentials as a `aws-secret` in kubernetes cluster. Then use those in the components.
-   1. You need credentials for an IAM user with SageMakerFullAccess. Apply them to k8s cluster.
-      Replace `AWS_ACCESS_KEY_IN_BASE64` and `AWS_SECRET_ACCESS_IN_BASE64`.
-      > Note: To get base64 string you can do `echo -n $AWS_ACCESS_KEY_ID | base64`
-      ```
-      cat <<EOF | kubectl apply -f -
-      apiVersion: v1
-      kind: Secret
-      metadata:
-        name: aws-secret
-        namespace: kubeflow
-      type: Opaque
-      data:
-        AWS_ACCESS_KEY_ID: <AWS_ACCESS_KEY_IN_BASE64>
-        AWS_SECRET_ACCESS_KEY: <AWS_SECRET_ACCESS_IN_BASE64>
-      EOF
-      ```
-   2. Use the stored `aws-secret` in pipeline code by adding this line to each component in your pipeline `.apply(use_aws_secret('aws-secret', 'AWS_ACCESS_KEY_ID', 'AWS_SECRET_ACCESS_KEY'))`   
-      [Kubeflow Document](https://www.kubeflow.org/docs/aws/pipeline/)  
-      [Example Code](https://github.com/kubeflow/pipelines/blob/master/samples/contrib/aws-samples/simple_train_pipeline/training-pipeline.py#L76) (uncomment this line)
+<details><summary>Standalone Kubeflow Pipelines Deployment</summary>
+<p>
 
-## Inputs to the pipeline
-
-### Role Input
-**Note:** Ignore this section if you plan to run [titanic-survival-prediction](https://github.com/kubeflow/pipelines/tree/master/samples/contrib/aws-samples/titanic-survival-prediction) example
-
-This role is used by SageMaker jobs created by the KFP to access the S3 buckets and other AWS resources.
-Run these commands to create the sagemaker-execution-role.   
-Note down the Role ARN. You need to give this Role ARN as input in pipeline.
-
-```
-TRUST="{ \"Version\": \"2012-10-17\", \"Statement\": [ { \"Effect\": \"Allow\", \"Principal\": { \"Service\": \"sagemaker.amazonaws.com\" }, \"Action\": \"sts:AssumeRole\" } ] }"
-aws iam create-role --role-name kfp-example-sagemaker-execution-role --assume-role-policy-document "$TRUST"
-aws iam attach-role-policy --role-name kfp-example-sagemaker-execution-role --policy-arn arn:aws:iam::aws:policy/AmazonSageMakerFullAccess
-aws iam attach-role-policy --role-name kfp-example-sagemaker-execution-role --policy-arn arn:aws:iam::aws:policy/AmazonS3FullAccess
-aws iam get-role --role-name kfp-example-sagemaker-execution-role --output text --query 'Role.Arn'
-
-# note down the Role ARN. 
-```
+1. [Install only Kubeflow Pipelines](https://docs.aws.amazon.com/sagemaker/latest/dg/setup.html#kubeflow-pipelines-standalone). 
```

Comment:
Install Kubeflow pipelines standalone by following the documentation on SageMaker developer guide